corehq/apps/ota/views.py

```diff
@@ -270,6 +271,9 @@ def get_restore_response(domain, couch_user, app_id=None, since=None, version='1
     if not restore_user:
         return HttpResponse('Could not find user', status=404), None
 
+    # successful sync, reset login attempts for mobile user
+    clear_login_attempts(as_user_obj or couch_user)
```

Comment:
This should definitely not clear the login attempts of the `as_user` since that is not the user who has authenticated.

Also, this will clear login attempts for users even when accessing this via the admin restore endpoint or anywhere else where we generate restores.